src/app/modules/vault/modules/organization-badge/organization-name-badge.component.ts

```diff
@@ -49,18 +50,7 @@ export class OrganizationNameBadgeComponent implements OnInit {
     const r = parseInt(color.substring(0, 2), 16); // hexToR
     const g = parseInt(color.substring(2, 4), 16); // hexToG
     const b = parseInt(color.substring(4, 6), 16); // hexToB
-
-    const uicolors = [r / 255, g / 255, b / 255];
-    const c = uicolors.map((c) => {
-      if (c <= 0.03928) {
-        return c / 12.92;
-      } else {
-        return Math.pow((c + 0.055) / 1.055, 2.4);
-      }
-    });
-
-    const L = 0.2126 * c[0] + 0.7152 * c[1] + 0.0722 * c[2];
-    return L > 0.179 ? "black !important" : "white !important";
+    return r * 0.299 + g * 0.587 + b * 0.114 > 186 ? "black !important" : "white !important";
```

Comment:
I think we  have two implementations for converting a string to color currently, with the avatar component. Any tech debt ticket to streamline this into a single implementation?

Reply:
Seems reasonable to add a note/subtask to that ticket